config/controllerrole.yaml

```diff
@@ -62,6 +62,13 @@ rules:
   - update
   - delete
   - watch
+- apiGroups:
+    - ""
+  resources:
+    - nodes
+  verbs:
+    - get
```

Comment:
Do we need `get` or is `list` suffice?

Reply:
we don't get individual nodes, we just list all of them and iterate so I think we can ditch the get